pkg/driver/distro/distro_test.go

```diff
@@ -233,3 +236,31 @@ func TestDiscoverDistro(t *testing.T) {
 		tCase.postFn()
 	}
 }
+
+func TestVerifyDownloadedSignature(t *testing.T) {
+	type testCase struct {
+		name                 string
+		downloadedDriverDest string
+		pubkey               string
+		valid                bool
+	}
+	testCases := []testCase{
+		{"valid local pubkey", "testdata/hello.txt", "testdata/key.pub", true},
+		// {"valid remote pubkey", "testdata/hello.txt", "https://raw.githubusercontent.com/LucaGuerra/falcoctl/8875a534256686a936213b8b8dc5bbf74b5febfc/internal/signature/testdata/key.pub", true},
+		// {"invalid remote pubkey", "testdata/hello.txt", "https://raw.githubusercontent.com/LucaGuerra/falcoctl/8875a534256686a936213b8b8dc5bbf74b5febfc/internal/signature/testdata/does_not_exist.pub", false},
```

Comment:
Note for the reviewer: These are disabled until the PR is merged, then, we can update the URL with the right repo and commit hash